mappings/net/minecraft/enchantment/EnchantmentHelper.mapping

```diff
@@ -31,9 +33,9 @@ CLASS net/minecraft/class_66 net/minecraft/enchantment/EnchantmentHelper
 		COMMENT Returns the increase in damage to the target due to the attacker's enchantments.
 		ARG 0 attacker
 		ARG 1 target
-	METHOD method_110 getEnchantmentResistance ([Lnet/minecraft/class_2056;Lnet/minecraft/class_1733;)I
-		COMMENT Returns the increase in protection from the damage source due to the armor stacks' enchantments.
-		ARG 0 armor
+	METHOD method_110 modifyProtection ([Lnet/minecraft/class_2056;Lnet/minecraft/class_1733;)I
+		COMMENT Returns the increase in protection from the damage source due to the item stacks' enchantments.
+		ARG 0 stacks
```

Comment:
This is a change back too space's name.

it is only ever called for armor. I think my name is better